webapp/medunited/care/view/pharmacy/Master.view.xml

```diff
@@ -0,0 +1,65 @@
+<mvc:View displayBlock="true"
+	controllerName="medunited.care.controller.pharmacy.Master" height="100%"
+	xmlns="sap.m" xmlns:f="sap.f" xmlns:mvc="sap.ui.core.mvc">
+	<f:DynamicPage id="pharmacyPageId"
+		toggleHeaderOnTitleClick="false">
+		<!-- DynamicPage Title -->
+		<f:title>
+			<f:DynamicPageTitle>
+				<f:heading>
+					<Title text="{i18n>Pharmacies}" />
+				</f:heading>
+				<f:actions>
+					<Button type="Emphasized" text="{i18n>Create}" press="onAdd" />
```

Comment:
Same thing here. If you use a key, make sure to have that key inside the i18n.properties file.
In this case, this key should be: 

Create=Anlegen